src/api-reference/user-provisioning/spend/v4.spend-user-extension-schema.md

```diff
@@ -34,8 +34,8 @@ Custom Data is the object used as the value of `customData` in the Spend User Ex
 
 |Name|Type|Format|Description|
 |---|---|---|---|
-|`id`|`string`|-|`custom1` - `custom22`, `orgUnit1` - `orgUnit6`|
-|`value`|`string`|-|Value of the custom field. For list = List Item Code.|
+|`id`|`string`|-|`custom1` - `custom22`, `orgUnit1` - `orgUnit6`.|
+|`value`|`string`|-|Value of the custom field. For list = List Item Code. Maximum length: 8 characters|
```

Comment:
Since the max length for `List Item Code` & `text value` is different, I don't have a good answer for this one. Let's remove this one for now.

Reply:
Sounds good!